docs/interacting-with-geth/rpc/ns-eth.md

```diff
@@ -180,7 +180,7 @@ Example:
 
 Returns a block header.
 
-#### Parameters
```

Comment:
It should be in the same level with usage and response, isn't it?
Perhaps change them as well?

Also, instead of changing it to plain text, maybe use **bold text**?

Reply:
thanks for the advice, please take another look.